index.html

```diff
@@ -1,68 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="/CSS/index.css">
-</head>
-<body>
-    <header>
-        <nav>
-            <ul>
-                <li><a href="#Skills">Skills</a></li>
-                <li><a href="#Projects">Projects</a></li>
-                <li><a href="#About">About</a></li>
-                <li><a href="#Experience">Experience</a></li>
-                <li><a href="#Connect">Connect</a></li>
-            </ul>
-        </nav>
-    </header>
-<main>
-    <h1 id="name">Anna Bazileeva</h1>
-    <section id="Skills">
-    <h2 id="Skills">Skills</h2>
-    <ul>
-        <li>Redrover school QA</li>
-        <li>Udemy Front-end developer</li>
-        <li><a href="https://classes.codethedream.org/course/intro-to-programming-v4/haumea">Code of Dream Intro-class</a></li>
-    </ul>
-    </section>
-    <section id="Projects">
-    <h2 id="Projects">Projects</h2>
-    </section>
-  <section id="About">
-<h2>About</h2>
-<p>I don't finish a previous ganemede course. But I hope to finish haumea class and go forward.</p>
-<img src="/IMG_B9F461A8A800-1.jpeg" width="100px"/>
-</section>
-<section id="Experience">
-<h2>Experience</h2>
-<div class="columns">
-<ul>
-    <li>2024</li>
-    <li>2022</li>
-    <li>2003</li>
-    <li>2000</li>
-</ul>
-    <ul> 
-    <li>TJMax</li>
-    <li>Scenthound</li>
-    <li>LLC</li>
-    <li>Military</li>
-    </ul>
-</div>
-</section>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="./CSS/index.css" />
+    </head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#Skills">Skills</a></li>
+                    <li><a href="#Projects">Projects</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Experience">Experience</a></li>
+                    <li><a href="#Connect">Connect</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <h1 id="name">Anna Bazileeva</h1>
+            <section id="About">
+                <h2>About</h2>
+                <p>
+                    I don't finish a previous ganemede course. But I hope to
+                    finish haumea class and go forward.
+                </p>
+                <img src="./CSS/img/IMG_B9F461A8A800-1.jpeg" width="100px" />
+            </section>
+            <section id="Skills">
+                <h2 id="Skills">Skills</h2>
```

Comment:
You're using the *"Skills"* `id` for two of your elements, the `section` and the `h2`. That should not happen, [ids should be unique](https://www.theodinproject.com/lessons/foundations-intro-to-css#id-selectors). Considering you have other sections that only use the `id` for the section, that's probably what you wanted here, right?


```suggestion
                <h2>Skills</h2>
```